app/assets/javascripts/short-form/ShortFormDataService.js.coffee

```diff
@@ -636,6 +636,7 @@ ShortFormDataService = (ListingDataService, ListingConstantsService, ListingPref
       when 'year'
         year >= 1900 && year <= validYear
 
+  # FIXME: this will fail in the year 2100
```

Comment:
this should be refactored, but I decided against it because in theory we will replace this logic before the year 2100